docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -54,6 +57,7 @@ spec:
 |Field | Description | Type | Required |
 | ---| ---| ---| --- |
 |``host`` | The host (domain name) of the server. Must be a valid subdomain as defined in RFC 1123, such as ``my-app`` or ``hello.example.com``. When using a wildcard domain like ``*.example.com`` the domain must be contained in double quotes.  The ``host`` value needs to be unique among all Ingress and VirtualServer resources. See also [Handling Host and Listener Collisions](/nginx-ingress-controller/configuration/handling-host-and-listener-collisions). | ``string`` | Yes |
+|``listener`` | Sets a custom HTTP and/or HTTPS listener. Must reference the name of a listener defined in a GlobalConfiguration resource | [listener](#virtualserverlistener) | No |
```

Comment:
we should probably mention that only two strings are valid i.e. `http` and `https`